src/components/VideoList/VideoList.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import styled from 'styled-components';
+import mockData from '../../youtube-videos-mock.json';
+
+const { items } = mockData;
+
+const VideoCardContainer = styled.div`
+  border: 1px solid #cccccc;
+  border-radius: 5px;
+  padding: 20px;
+  background-color: #e7e7fc;
+`;
```

Comment:
I recommend using the structure that the base project had.
Have all business logic and rendering on `VideoList.component.jsx`
Have all the styling on `VideoList.styles.js`